docs/modules/publisher-commandline.md

```diff
@@ -104,8 +107,8 @@ The following list of module parametrization settings can be provided either as
                                       Default: {00:00:00} meaning heartbeat is disabled
 
             DefaultSkipFirst=VALUE
-                                      Default for Skip First published node setting. When set to true the 
-                                      publisher will not send as telemetry the initial datachange event received
+                                      Default for SkipFirst published node setting. When set to true OPC Publisher 
```

Comment:
do we support SkipFirst in 2.8.2? I have not found any code for this.

Reply:
Doesn't seem so, I also cannot find any logic around it in the code. In `2.8.1` and `2.8.0` there was a todo item in the code. Seems like we have added the propagation of the value through job converter, but there is no actual logic to do the skipping.

Reply:
How should we handle this? Remove from the doc and remove from code?

Reply:
Command line arguments related to `SkipFirst` are commented out in #1612.

Reply:
removed